src/taoInstance.js

```diff
@@ -350,6 +352,40 @@ module.exports = function taoInstanceFactory(rootDir = '', quiet = true, wwwUser
                 execed.stderr.pipe(process.stderr);
                 execed.on('exit', code => code === 0 ? resolve() : reject( new Error('Something went wrong in the translation generation')));
             });
+        },
+
+        /**
+        * Update version in extension manifest
+        *
+        * @param {String} manifestPath - the path to the extension manifest
+        * @param {String} version - extension version
+        * @return {Promise}
+        */
+        updateVersion(manifestPath = '', version) {
+            return new Promise((resolve, reject) => {
+                fs.readFile(manifestPath, 'utf8', function (err, manifestContent) {
+                    if (err) {
+                        return reject(err);
+                    }
+
+                    if (!versionPropRegex.test(manifestContent)) {
+                        reject(new Error('Can not extract version from manifest file'));
+                    }
+
+                    const updatedManifestContent = manifestContent.replace(
+                        versionPropRegex,
+                        (match, versionMatch) => match.replace(versionMatch, `'${version}'`)
+                    );
+
+                    fs.writeFile(manifestPath, updatedManifestContent, 'utf8', function (err) {
+                        if (err) {
+                            return reject(err);
+                        }
+
+                        resolve();
+                    });
+                });
+            });
```

Comment:
you have the package `fs-extra` available or you can use `util.promisify` so you can use promise version of `fs`